events/bingo/bingoBossApp/ui/js/bingoBossApp_ui.js

```diff
@@ -0,0 +1,67 @@
+//
+//  bingoBossApp_ui.js
+//
+//  Created by Zach Fox on 2019-02-15
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+/* globals $ document EventBridge setTimeout */

```

Comment:
We can add more of these common things we always put in the globals in the main selling file

Reply:
is that something i can do in this PR?

setting `setTimeout` and `setInterval` as globals for all js files would be dangerous, since they're not globals in App JS scripts. the other three here might make sense...

Reply:
Yeah, if you open up the .eslintrc file, I've just been adding things to the list.  If you do it, can you add module?

Reply:
done